client_test/conftest.py

```diff
@@ -574,6 +576,36 @@ async def QueueGet(self, stream):
         await stream.recv_message()
         await stream.send_message(api_pb2.QueueGetResponse(values=[self.queue.pop(0)]))
 
+    ### Sandbox
+
+    async def SandboxCreate(self, stream):
```

Comment:
Might be useful to use the new `servicer.intercept()` helper here – that way you can put the fixture logic in the test itself. See an example: https://github.com/modal-labs/modal-client/blob/main/client_test/cli_test.py#L378 (haven't used it myself yet, @freider built it)